### DISCUSSION TITLE
remove or change background color hover on FormBuilderCheckbox (Web)

### DISCUSSION THREAD

**Discussion by u-sour** [danvick/flutter_form_builder#1329] [Q&A]
hi,everyone, how can i remove or change background color hover on FormBuilderCheckbox ? thank in advance

https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/assets/145416126/ab94db6e-1057-4d28-91fd-3e6c11bee253

**Comment by deandreamatias** [comment]
Hi,
This still happens? I think that with decoration properties can disabled this. Is a behavior of Material components

**Reply by u-sour** [reply]
Yeah, no problem with this. thanks.
